arches/app/media/js/views/components/cards/photo-gallery-card.js

```diff
@@ -59,6 +60,10 @@ define([
             return mfs;
         });
 
+        this.acceptedFiles = ko.computed(function(){
+            return self.card.widgets().find(widget=>widget.node_id() === self.fileListNodeId)?.config.acceptedFiles() || 'All Formats Accepted';
```

Comment:
This is a really nice improvement over the `--`. Could you add a translation string to javascript.htm for 'All Formats Accepted'?

Reply:
Translation string added - hopefully I've gone about this the right way; I was using the pattern set in search.js